danieldupriest/reimbursinator
# Delete report api

Testing Steps:

1. Navigate to POST Login to Account tab, and in the "body" tab change the `user` field to `email`, and login with `user@user.com`, password `user12345`
2. Copy the auth key sent back after login and paste it into the Auth tab of the "Authenticated" folder (click the three dots to view "edit"). 
3. Run the "List reports" to get the report_pk of the "delete reports" I created for testing
4. Run the "Delete report" command with the following url: `localhost:8000/api/v1/report/[report_pk]`
5. After the JSON "Deleted report [report_pk]" is printed, run docker and log into user@user.com. Click edit reports, and poof! Your report is now successfully deleted!

NOTE: Also added the link for per diem rates - should be visible in any of the reports after "REPORT CREATION"
